securedrop_client/gui/widgets.py

```diff
@@ -2007,351 +2009,538 @@ def _on_left_click(self):
-        self.generic_error.setObjectName('gener_error')
-        generic_error_layout = QHBoxLayout()
-        self.generic_error.setLayout(generic_error_layout)
-        self.error_status_code = SecureQLabel()
-        generic_error_message = SecureQLabel(_('See your administrator for help.'))
-        generic_error_message.setWordWrap(True)
-        generic_error_layout.addWidget(self.error_status_code)
-        generic_error_layout.addWidget(generic_error_message)
-
-        # Insert USB Device Form
-        self.insert_usb_form = QWidget()
-        self.insert_usb_form.setObjectName('insert_usb_form')
-        usb_form_layout = QVBoxLayout()
-        self.insert_usb_form.setLayout(usb_form_layout)
-        self.usb_error_message = SecureQLabel(_(
-            'Either the drive is not LUKS-encrypted, or there is something '
-            'else wrong with it.'))
-        self.usb_error_message.setWordWrap(True)
-        usb_instructions = SecureQLabel(_(
+        self.error_status = ''  # Hold onto the error status we receive from the Export VM
+
+        # Connect controller signals to slots
+        self.controller.export.preflight_check_call_success.connect(self._on_preflight_success)
+        self.controller.export.preflight_check_call_failure.connect(self._on_preflight_failure)
+        self.controller.export.export_usb_call_success.connect(self._on_export_success)
+        self.controller.export.export_usb_call_failure.connect(self._on_export_failure)
+
+        # Connect parent signals to slots
+        self.continue_button.setEnabled(False)
+
+        self.header_icon.update_image('savetodisk.svg', QSize(64, 64))
+
+        # Dialog content
+        self.starting_header = _(
+            'Preparing to export:'
+            '<br />'
+            '<span style="font-weight:normal">{}</span>'.format(self.file_name))
+        self.insert_usb_header = _('Insert encrypted USB drive')
+        self.passphrase_header = _('Enter passphrase for USB drive')
+        self.success_header = _('Export successful')
+        self.error_header = _('Unable to export')
+        self.starting_message = _(
+            '<h2>Proceed with caution when exporting files</h2>'
+            '<b>Malware</b>'
+            '<br />'
+            'This workstation lets you open documents securely. If you open documents on another '
+            'computer, any embedded malware may spread to your computer or network. If you are '
+            'unsure how to manage this risk, please print the document, or contact your '
+            'administrator.'
+            '<br /><br />'
+            '<b>Anonymity</b>'
+            '<br />'
+            'Documents submitted by sources may contain information or hidden metadata that '
+            'identifies who they are. To protect your sources, please consider redacting documents '
+            'before working with them on network-connected computers.')
+        self.exporting_message = _('Exporting: {}'.format(self.file_name))
```

Comment:
is this unused? I was looking through these attributes to trace where `file_name` gets used to check for potential injection issues and can't find where this attribute `exporting_message` is used

Reply:
ah, good find. I can remove this line.

Reply:
`exporting_message` still unused?